packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md

```diff
@@ -10,46 +10,46 @@ This guide covers how to upgrade the Okta Sign-In Widget, which depends on wheth
 
 **Learning outcomes**
 
-* Understand the benefits of keeping your widget up to date.
-* Know how to obtain the latest version of the widget.
 * Update your widget based on your user deployment model.
 
 **What you need**
 
-[Latest available widget release](https://github.com/okta/okta-signin-widget/releases)
+* An [Identity Engine-upgraded Okta org](/docs/guides/oie-upgrade-overview/)
+* An existing app that uses the Okta Sign-In Widget
+* [Latest available widget release](https://github.com/okta/okta-signin-widget/releases)
 
 ---
 
 ## About the widget
 
 The widget is a JavaScript library that gives you a full-featured and customizable sign-in experience that you can use to authenticate users of web and mobile applications.
 
-The widget is used on Okta's default sign-in page to start an Okta SSO session and to set the Okta [session cookie](/docs/guides/session-cookie/) in the web browser. The widget can also perform a complete [OpenID Connect (OIDC)](/docs/reference/api/oidc/) flow and/or integrate with [external Identity Providers](/docs/concepts/identity-providers/).
+The widget is used on Okta's default sign-in page to start an Okta SSO session and to set the Okta [session cookie](/docs/guides/session-cookie/) in the web browser. The widget can also perform a complete [OpenID Connect (OIDC)](/docs/concepts/oauth-openid/) flow and/or integrate with [external Identity Providers](/docs/concepts/identity-providers/).
 
 This article teaches you how to upgrade the widget when it is used in any of the following ways:
 
 * Okta-hosted sign-in page (default): Okta provides a sign-in page that is available at your [org's URL](/docs/concepts/okta-organizations/). By default, a user who signs in on this page is redirected to the Okta user dashboard.
 * Okta-hosted sign-in page (customizable): Okta provides a hosted sign-in page that you can customize and make available under a custom domain that is a subdomain of your company's top-level domain.
 * Embedded (self-hosted): You can embed the widget directly into your application.
 
-After you've completed the widget upgrade and you want to take advantage of the new features in Okta Identity Engine, you can [configure your embedded widget](https://github.com/okta/okta-signin-widget/blob/master/README.md#embedded-self-hosted) to use the new Identity Engine Authentication features.
+After you've completed the widget upgrade, review [Okta Identity Engine overview](/docs/guides/oie-intro/) to take advantage of new features in Identity Engine.
 
 ## Best practice for widget maintenance
 
 For best practices, keep your [widget](https://github.com/okta/okta-signin-widget/blob/master/README.md) up to date. This maintenance is essential so that you can use the latest Identity Engine features and benefit from ongoing improvements to the codebase.
```

Comment:
Perhaps this should point to the releases page of the repo rather than the readme? 
```suggestion
For best practices, keep your [widget](https://github.com/okta/okta-signin-widget/releases) up to date. This maintenance is essential so that you can use the latest Identity Engine features and benefit from ongoing improvements to the codebase.
```